hi-cache-server/src/com/hh/cache/process/server/CommitDiskThread.java

```diff
@@ -21,7 +21,21 @@
     public static FileOutputStream cacheWriter;
     public static ConcurrentLinkedQueue<byte[]> content = new ConcurrentLinkedQueue();
     private static final File cacheFile = new File(StartApp.config.getConfig("data-path") + "/cache.log").getAbsoluteFile();
-
+    private static final int MAX_SIZE_CONTENT_COMMIT_TO_DISK;
+    static {
+        int config = 1000;
+        try {
+            config = new Integer(StartApp.config.getConfig("max_size_content_commit_to_disk"));
```

Comment:
Should we use Constants instead of StartApp? Cause I do think from now on, we're gonna use it.